packages/documentation-framework/scripts/webpack/webpack.base.config.js

```diff
@@ -117,6 +117,7 @@ module.exports = (_env, argv) => {
         'client-styles': path.resolve(process.cwd(), 'patternfly-docs/patternfly-docs.css.js'),
         './routes-client': path.resolve(process.cwd(), 'patternfly-docs/patternfly-docs.routes.js'),
         './routes-generated': path.resolve(process.cwd(), 'patternfly-docs/generated/index.js'),
+        '@patternfly/react-core/next': '@patternfly/react-core/dist/js/next/index.js',
```

Comment:
why is this needed?

Reply:
It shouldn't be after the next path exists in patternfly-react. This PR wasn't quite ready to be reviewed as it depends on the version of react-core that has the "next" path.

Reply:
Spoke with @jschuler in slack about this and we decided to keep this around until webpack is updated to v5 for the documentation-framework.